integr8ly/integreatly-operator
# Implement logic to allow 3scale monitoring to be enabled

## JIRA
[MGDAPI-49](https://issues.redhat.com/browse/MGDAPI-49)

## Description
3scale 2.9 ships with monitoring resources (new alerts and dashboards) that are disabled by default. This PR introduces allows us to enable monitoring (via a flag in the API Manager CR) when required. For now, the flag is set to false. The 3scale operator version has also been updated to allow this change to be made.

## Steps to verify
- We need to ensure that once we do enable the monitoring flag in the API Manager CR, the correct alerts and dashboards are created. Using this branch, set [this line](https://github.com/integr8ly/integreatly-operator/pull/1200/files#diff-c99ff6dd3de866743c817f7b1dc230b4875e42eba21e5768f34fd92b9f873e32R909) to `true` and complete an install of RHMI
- Once the install is complete, ensure that all of the new 3scale alerts/dashboards were created in Prometheus and Grafana. A list of both can be found [here](https://gist.github.com/obrienrobert/b390ad39d0c4f63d930a094a8ef4a594)
- Ensure that none of the new alerts are firing after install

## Type of Change



## Checklist
